spikes/oms-log-analytics/README.md

```diff
@@ -4,59 +4,60 @@ Prometheus is a popular open source metric monitoring solution and is a part of
 
 Reference Documentation [here](https://docs.microsoft.com/en-us/azure/azure-monitor/containers/container-insights-prometheus-integration).
 
-## Steps to Scrape Metrics into Log Analytics
+## Installation on AKS Clusters
 
-1. Save the Workspace ID and the key for log Analytics work space from the Azure dashboard
-2. If deploying to a AKS cluster, check whether you already have OMS Agent running.
-   1. Check if `omsagent` Pod is running: `kubectl get pods -A | grep omsagent`
-   2. If `omsagent` is running then only apply the config file: `kubectl apply -f 3-container-azm-ms-agentconfig.yaml`
-   3. Skip to Step 6
-3. To use OMS Agent Daemonset and deployment:
-   1. Create the log analytics secrets with kubectl:
+For non-AKS clusters please see [non-aks/README.md](non-aks/README.md).
 
-    ```bash
+For installing on AKS clusters, we recommend enabling Azure Monitor for Containers through your Infrastructure as Code solution.
+Please see, [Enable Container Insights](https://docs.microsoft.com/en-us/azure/azure-monitor/containers/container-insights-onboard) for instructions on enabling Azure Monitor for Containers for your cluster.
 
-    kubectl create secret generic omsagent-secret -n kube-system \
-                --from-literal=KEY=$(az monitor log-analytics workspace get-shared-keys \
-                -g LOG_ANALYTICS_GROUP -n LOG_ANALYTICS_NAME --query primarySharedKey -o tsv) \
-                --from-literal=WSID=$(az monitor log-analytics workspace show \
-                -g ngsa-fb-test-test-rg -n ngsa-fb-test-log --query customerId -o tsv)
-    # If you don't have az cli access, replace the $(az monitor...) portion with actual values
-    
-    ```
+For more detail on what is created, please see the yaml files in non-aks.
```

Comment:
AKS should probably be capitalized (and maybe YAML too, since it's an acronym).

```suggestion
For more detail on what is created, please see the YAML files in non-AKS.
```

Reply:
Made the change. For the "non-aks" one, I put the folder name in quotes, because it is referring to a folder in the same location